python/lsst/obs/subaru/isr.py

```diff
@@ -247,6 +247,11 @@ def runDataRef(self, sensorRef):
                 self.saturationDetection(ccdExposure, amp)
             if self.config.doSuspect and not badAmp:
                 self.suspectDetection(ccdExposure, amp)
+            ampMask = afwImage.MaskU(ccdExposure.getMaskedImage().getMask(), amp.getRawDataBBox(),
+                                     afwImage.PARENT)
+            maskVal = ampMask.getPlaneBitMask([self.config.saturatedMaskName, self.config.suspectMaskName])
+            if numpy.all(ampMask.getArray() & maskVal > 0):
+                badAmp = True
             if self.config.doOverscan and not badAmp:
```

Comment:
Thanks to @RobertLuptonTheGood for a detailed explanation of the bitwise operator...I now understand what's going on!  He also suggests it may be safer to put parentheses around this operation: `numpy.all((ampMask.getArray() & maskVal) > 0)`